src/player.cpp

```diff
@@ -6830,6 +6859,27 @@ std::list<item> player::use_charges( const itype_id& what, long qty )
         use_fire( qty );
         return res;
 
+    } else if( what == "breath" ) {
+        auto rebreathers = all_items_with_flag( "SCUBA" );
+        for( auto &i : rebreathers ) {
+            auto reb = charges_of( i->typeId(), ( long )ceil( qty * 0.5 ) );
+            if( reb > 0 ) {
+                auto found = use_charges( i->typeId(), reb );
+                res.splice( res.end(), found );
+                qty -= std::min( qty, long( reb / 0.5 ) );
+            }
+        }
+
+        auto airsupply = all_items_with_flag( "AIR_SUPPLY" );
+        for( auto &i : airsupply ) {
+            auto air = charges_of( i->typeId(), ( long )ceil( qty ) );
```

Comment:
Calling `ceil` on a `long` is pointless (it does rounding to integers, but a `long` is already an integer).